napari/_vispy/experimental/texture_atlas.py

```diff
@@ -273,6 +273,9 @@ def add_tile(self, octree_chunk: OctreeChunk) -> Optional[AtlasTile]:
 
         assert isinstance(data, np.ndarray)
 
+        # Convert all data to float32
+        data = data.astype(np.float32)
```

Comment:
I think we need to do a conversion to float32. In current vispy code the conversion is done during normalizing according to the clims. Right now we don't have the clim info available to us here, but we could change that. Not sure though after the `0.7` vispy release what the best approach would be though